### ISSUE TITLE
Support ngx_http_proxy_connect_module

### ISSUE THREAD

**Issue by seonghobae** [cryptofuture/nginx-hda-bundle#37]
Hello,
I want to know how can I implement ngx_http_proxy_connect_module in this ppa.
I tried to compile ngx_http_proxy_connect_module as dynamic module, however that returns 'not binary compatatible' even I tried to add options in ```./configure```.

Please give some tips.
https://github.com/chobits/ngx_http_proxy_connect_module

**Comment by cryptofuture** [comment]
You could get idea from here: https://github.com/cryptofuture/nginx-hda-bundle/commit/fbd36660c77b98c155a773b37ff0eb09d90bd9ef
Basicall you need to edit control, rules files and add submodule (this in case module is building dynamic without any issues)

**Comment by seonghobae** [comment]
Oh, that means, if I add some GitHub links in there and push them, you can update the PPA repository?

**Comment by cryptofuture** [comment]
If you add module correctly surely, I could update a PPA.
https://github.com/cryptofuture/nginx-hda-bundle#i-want-to-add-my-or-someones-nginx-module-to-the-ppa

**Comment by seonghobae** [comment]
Okay... if I can not, some donation is helpful, right?

**Comment by cryptofuture** [comment]
@seonghobae Yes, just don't expect me start to do it right away, through I could try at night for example, as I'm still busy full-time.
Etherway, I suggest you to just do it yourself, as it could get you familiar with how nginx module system work on the way.
Usually I trying to add new modules with every new nginx release, however in the last time, I'm a bit too busy with another work. So mostly do it on saturday/sundays after release.

**Comment by seonghobae** [comment]
I made a small donation for you, and I hope so this project grow up more.

**Comment by cryptofuture** [comment]
@seonghobae Thx, every little bit helps.

**Comment by cryptofuture** [comment]
close in 375e1db1d243547b3fcb344aa783d8c2d51fa97a
@seonghobae Please check `nginx-module-http-proxy-connect` as it builds

**Comment by seonghobae** [comment]
Install and load correctly, however, I have to check that runs appropriately.
